docs/source/contributing.rst

```diff
@@ -33,17 +34,101 @@ If you're unsure about any of these points, please open the Pull Request anyhow
 Pull Request Process
 --------------------
 
-After you open the Pull Request, there will probably be some discussion in the comments' field of the request itself.
+After you open the Pull Request, there will probably be some discussion in the comments field of the request itself.
 
 Once the Pull Request is ready to merge, it will first be merged into our internal git system for in-house automated testing.
```

Comment:
Optionally, we can put phase "in-house" into quotation marks as it's our "naming".